content/docs/guides/aws-lambda.md

```diff
@@ -0,0 +1,148 @@
+    npm install -g serverless
+    ```
+
+2. Create a `my-lambda` project and change to that directory.
+
+    ```bash
+    mkdir my-lambda
+    cd my-lambda
+    ```
+
+3. Run the **serverless** command to create a serverless project. When prompted, select the **AWS - Node.js - Starter** project and provide your AWS account credentials, which include your AWS Access Key Id and AWS Secret Access Key. The process creates an `aws-node-project` directory.
+
+    ```bash
+     serverless
+    ```
+
+4. Install the `node-postgres` package, which you will use to connect to the database.
+
+    ```bash
+    npm install pg
+    ```
+
+5. Create a new file named `user.js` and add the following code:
+
+    ```js
+    'use strict';
+
+    const { Client } = require('pg');
+
+    module.exports.getAllUsers = async () => {
+     var client = new Client(process.env.DATABASE_URL);
+     client.connect();
+     var { rows } = await client.query('SELECT * from users);
+     return {
+       statusCode: 200,
+       body: JSON.stringify({
+         data: rows,
+       }),
+     };
+    };
+    ```
+
+    The code above exports the function `getAllUsers`, which retrieves all rows from the `users` table and returns them as a `JSON` object in the `HTTP` response body.
+
+    The function uses the `pg` library to connect to the Neon database using the `Client` class and the database connection URL that is stored in the `DATABASE_URL` environment variable. The function calls the connect method on the `Client` instance to establish a connection to the database, and uses the query method to execute a `SELECT` statement that retrieves all rows from the `users` table.
+
+    The query method returns a `Promise` that resolves to an object containing the rows retrieved by the `SELECT` statement, which the function parses to retrieve the `rows` property. Finally, the function returns an `HTTP` response with a status code of 200 and a body that contains a `JSON` object with a single `data` property, which is set to the value of the rows variable.
+
+6. Add the `DATABASE_URL` environment variable and the function definition to the `serverless.yml` file, which is located in your `aws-node-project` directory.
+
+    You can copy the connection string from the Neon Console, and add the `DATABASE_URL` under `environment`. Add `sslmode=require` to enable SSL. The `sslmode=require` option tells PostgreSQL to use SSL encryption and verify the server's certificate.
+  
+    ```yaml
+    org: aws-user
+    app: lambda-example
+    service: nodejs
+    frameworkVersion: '3'
+
+    provider:
+     name: aws
+     runtime: nodejs14.x
+     region: eu-west-1
+     environment:
+       DATABASE_URL: postgres://<USER>:<PASSWORD>@<HOST>/<DATABASE>?sslmode=require
+
+    functions:
+     getAllUsers:
+       handler: users.getAllUsers
+       events:
+         - httpApi:
+             path: /users
+             method: get
+    ```
+
+7. Deploy your function using the following command:
```

Comment:
The serverless deploy failed for me. Do I require a Serverless Framework account? I got this message:
Warning: Invalid configuration encountered
  at 'org': must match pattern "^[a-z0-9]{5,39}$"
Learn more about configuration validation here: http://slss.io/configuration-validation
Deploying nodejs to stage dev (eu-west-1)
✖ Stack nodejs-dev failed to deploy (0s)
Environment: linux, node 16.17.1, framework 3.26.0, plugin 6.2.2, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
You are not currently logged in. To log in, use: $ serverless login

Reply:
Issue addressed.